### PR TITLE
871372: Added UG for Maui SfChat GettingStarted

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG for .NET Maui SfChat GettingStarted

### CODE REVIEW

File: MAUI/Chat/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+         }
+       }
+    }
+    
+{% endhighlight %} 
+{% endtabs %}
+
+## Creating ViewModel for the Chat
+
+The `SfChat` control is data-bound and displays a collection of messages exchanged between users. Hence, messages should be created and binded to the control.
```
Comment (PiruthivirajM) [inline MAUI/Chat/getting-started.md:84]: .... and **bound** to the control.

File: MAUI/Chat/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+Create a simple message collection as shown in the following code example in a new class file. Save it as `GettingStartedViewModel.cs` file.
```
Comment (PiruthivirajM) [inline MAUI/Chat/getting-started.md:86]: save it as `ViewModel.cs` file

File: MAUI/Chat/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+{% endhighlight %}
+
+{% highlight c# hl_lines="1"%}
+ this.sfChat.ShowTimeBreak = true;
+         
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Chat time break](images/getting-started/maui-chat-timebreak.png)
```
Comment (PiruthivirajM) [inline MAUI/Chat/getting-started.md:261]: ![Time break support in .NET MAUI Chat]

File: MAUI/Chat/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+        this.sfChat.Messages = viewModel.Messages; 
+        this.sfChat.CurrentUser = viewModel.CurrentUser; 
+        this.Content = sfChat; 
+     } 
+  } 
+    
+{% endhighlight %} 
+{% endtabs %}
+
+![.NET MAUI Chat getting started](images/getting-started/maui-chat-getting-started.png)
```
Comment (PiruthivirajM) [inline MAUI/Chat/getting-started.md:242]: ![Chat control in .NET MAUI platform]

File: MAUI/Chat/getting-started.md (hunk 1 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/getting-started.md]: Changes the sentence

File: MAUI/Chat/getting-started.md (hunk 2 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/getting-started.md]: Addressed the concerns provided

File: MAUI/Chat/getting-started.md (hunk 3 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/getting-started.md:261]: Addressed the concerns

File: MAUI/Chat/getting-started.md (hunk 4 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/getting-started.md:242]: Renamed as per the requirement

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**